LEGO1/mxomni.cpp

```diff
@@ -42,10 +42,16 @@ void MxOmni::Init()
   m_unk64 = NULL;
 }
 
-// OFFSET: LEGO1 0x100b0090 STUB
-void MxOmni::vtable0x20()
+// OFFSET: LEGO1 0x100b0090
+long MxOmni::Start(MxDSAction* ds)
 {
-  // TODO
+  MxResult result = -1;
```

Comment:
Please use the appropriate constants for `MxResult` values (`SUCCESS` and `FAILURE`) (several places in this PR)